apps/documentation/docs/contribute-to-luos.mdx

```diff
@@ -0,0 +1,126 @@
+---
+custom_edit_url: null
+---
+
+import Image from '@site/src/components/Image.js';
+import IconExternalLink from '@theme/Icon/ExternalLink';
+import Form from '/src/components/typeform/index.js';
+
+
+# Contribute to Luos
+
+## Contribution
+
+As an open source project, Luos is open to contribution. It means that anyone can actively contribute in improving the project by reporting bugs or typos, or working on issues and pull requests.
+
+This page explains how to begin you journey through contribution.
+
+## Luos's contribution guidelines
+
+The first step is to read [Luos's contribution guidelines](https://github.com/Luos-io/luos_engine/blob/main/CONTRIBUTING.md) to discover the basics and the different layers of contribution.
+
+
+## Issues and PR
+
+After this reading, the next step is to get familiar with the [workflow](https://docs.github.com/en/get-started/quickstart/github-flow) we use with GitHub. Basically, it is about GitHub's **issues** and **pull requests**.
+
+### Issues
+
+:::info
+A <a href="https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues" target ="_blank">GitHub issue</a> is like a ticket used as a tracking tool, associated to a repository. It helps tracking bugs, feature requests, feedbacks, etc., and provides the details of the tracked subject. It is also used to discuss of the subject with other contributors.
+:::
+
+As a contributor, you will sometimes open new issues, for example if you spot a typo, want to report a bug, or request a new feature. 
+
+But you will also be able to contribute by commenting in existing issues by bringing some inputs and discussing bug resolving or feature implementing with other contributors.
+
+#### Checklists
+
+**Before creating a new issue:**
+
+- [ ] Verify what you want to talk about: if you are reporting a bug, are you sure it is an actual bug and not a particular way of using Luos that doesn't suite you in a particular case? 
+- [ ] If you want to request a feature, did you check that there is no way to do what you want to do? Do not hesitate to search in the <a href="https://docs.luos.io/docs" target ="_blank">documentation</a> first and <a href="https://discord.gg/luos" target ="_blank">ask questions on the Discord</a> before to rush in the creation of a new issue.
```

Comment:
Change link from : 'https://docs.luos.io/docs' to 'https://www.luos.io/docs/luos-technology'